From 998cecb9b19981c8f1c873f0910bb5972d4a2b4a Mon Sep 17 00:00:00 2001
From: xyyx <xyyx@mail.ru>
Date: Thu, 16 Aug 2018 06:33:26 +0000
Subject: [PATCH 3/5] LockscreenCharging: squashed (3/3)

based on:
[1/2] Settings: show more battery info on lockscreen when charging by yank555-lu

Change-Id: I4919cc59858896f07294953d5f7ae330f0d92c3f
Signed-off-by: DennySPb <dennyspb@gmail.com>
---
 res/values/cm_strings.xml    | 4 ++++
 res/xml/display_settings.xml | 7 +++++++
 2 files changed, 11 insertions(+)

diff --git a/res/values/cm_strings.xml b/res/values/cm_strings.xml
index 90cc5c23af..d1200582f1 100644
--- a/res/values/cm_strings.xml
+++ b/res/values/cm_strings.xml
@@ -166,4 +166,8 @@
 
     <!-- Screenshot shutter -->
     <string name="screenshot_shutter_sound_title">Screenshot shutter sound</string>
+
+    <!-- Lockscreen battery info indicator  -->
+    <string name="lockscreen_battery_info_title">Lockscreen charging info</string>
+    <string name="lockscreen_battery_info_summary">Display negociated charger max current and voltage and battery temperature on lockscreen while charging</string>
 </resources>
diff --git a/res/xml/display_settings.xml b/res/xml/display_settings.xml
index 5398d2d51e..62dc4cf47a 100644
--- a/res/xml/display_settings.xml
+++ b/res/xml/display_settings.xml
@@ -165,6 +165,13 @@
         android:title="@string/tap_to_wake"
         android:summary="@string/tap_to_wake_summary" />
 
+    <!-- lockscreen charging -->
+    <lineageos.preference.SystemSettingSwitchPreference
+        android:key="lockscreen_battery_info"
+        android:title="@string/lockscreen_battery_info_title"
+        android:summary="@string/lockscreen_battery_info_summary"
+        android:defaultValue="true" />
+
     <!-- Prevent accidental wake-up -->
     <lineageos.preference.LineageSystemSettingSwitchPreference
         android:key="proximity_on_wake"
-- 
2.31.1.606.gdf6c4f722c

